drivers/usb/device/usb_dc_stm32.c

```diff
@@ -50,6 +50,7 @@ LOG_MODULE_REGISTER(usb_dc_stm32);
 #endif
 
 #define USB_BASE_ADDRESS	DT_INST_REG_ADDR(0)
+#define USB_DEVICE_REG	((USB_OTG_DeviceTypeDef *)(USB_BASE_ADDRESS + USB_OTG_DEVICE_BASE))
```

Comment:
This seems instance specific.

Reply:
Okay, removed the USB_DEVICE_REG, but unfortunately the USB_CORE_REG will have to stay for now, because there is no implementation I could find. Only the USB_CoreReset, but it is static, so not exported.